Sources/PointFree/Episode/Show.swift

```diff
@@ -62,7 +62,27 @@ let episodeView = View<(Database.User?, Stripe.Subscription.Status?, Episode)> {
           )
         ]
       )
-      ])
+      ]),
+
+    script(
+      """
+      var hasPlayed = false;
```

Comment:
I don't enable the shortcut till someone's started to play the video. In Chrome with auto-play, the shortcut is enabled immediately. For Safari, it's lazy, so folks don't get spacebar highjacked till they start watching.